.babelrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (api) => ({
+  presets: ['@react-bootstrap', '@babel/typescript'],
+  plugins: [
+    api.env() === 'esm' && ['babel-plugin-add-import-extension', { 'extension': 'js' }],
```

Comment:
Using full path with extensions is required by ESM loader spec. The plugin does exactly that.

```js
import isWindow from './isWindow';
```

becomes 

```js
import isWindow from './isWindow.js';
```

Folder imports like:

```js
import { something } from './folder';
```

become

```js
import { something } from './folder/index.js';
```